cypress/integration/recipe_detail_spec.js

```diff
@@ -0,0 +1,354 @@
+          "name": "Spicy Arrabiata Penne",
+          "category": "Vegetarian",
+          "area": "Italian",
+          "doneDate": "22/6/2020",
+          "tags": ["Pasta", "Curry"]
+        }];
+        localStorage.setItem('doneRecipes', JSON.stringify(doneRecipes));
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').should('not.be.visible');
+  });
+
+  it('verifica se botão de iniciar receita não é visível na tela de detalhes de uma bebida', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        const doneRecipes = [{
+          "type": "bebida",
+          "id": "178319",
+          "image": "https://www.thecocktaildb.com/images/media/drink/zvsre31572902738.jpg",
+          "name": "Aquamarine",
+          "category": "Cocktail",
+          "alcoholicOrNot": "Alcoholic",
+          "doneDate": "23/6/2020",
+          "tags": null
+        }];
+        localStorage.setItem('doneRecipes', JSON.stringify(doneRecipes));
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').should('not.be.visible');
+  });
+});
+
+describe('Caso a receita tenha sido iniciada mas não finalizada, o texto do botão deve ser "Continuar Receita"', () => {
+  it('verifica botão de Continuar Receita na tela de detalhes de uma comida', () => {
+    cy.visit('http://localhost:3000/comidas/52771', {
+      onBeforeLoad(win) {
+        const inProgressRecipes = [52771];
+        localStorage.setItem('inProgressRecipes', JSON.stringify(inProgressRecipes));
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').contains('Continuar Receita');
+  });
+
+  it('verifica botão de Continuar Receita na tela de detalhes de uma bebida', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        const inProgressRecipes = [178319];
+        localStorage.setItem('inProgressRecipes', JSON.stringify(inProgressRecipes));
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').contains('Continuar Receita');
+  });
+});
+
+describe('Quando o botão "Iniciar Receita" for clicado, a rota deve mudar para a tela de receita em processo', () => {
+  it('redireciona para tela de receita (de uma comida) em progresso', () => {
+    cy.visit('http://localhost:3000/comidas/52771', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').click();
+    cy.location('href').should('include', '/in-progress');
+  });
+
+  it('redireciona para tela de receita (de uma bebida) em progresso', () => {
+    cy.visit('http://localhost:3000/bebidas/178319', {
+      onBeforeLoad(win) {
+        win.fetch = fetchMock;
+      },
+    });
+
+    cy.get('[data-testid="start-recipe-btn"]').click();
+    cy.location('href').should('include', '/in-progress');
```

Comment:
Ao verificar se a receita está em progresso, seria bom também garantir que o restante do path está correto, o que também garante que o ID se manteve.

```suggestion
    cy.location().should((loc) => expect(loc.pathname).to.eq('/bebidas/178319/in-progress'));
```